lib/connectionConfig/connectionConfig.ts

```diff
@@ -83,9 +84,9 @@ export namespace ConnectionConfig {
    * @returns {ConnectionConfig} ConnectionConfig
    */
   export function create(connectionString: string, path?: string): ConnectionConfig {
-    if (!connectionString || (connectionString && typeof connectionString !== "string")) {
-      throw new Error("'connectionString' is a required parameter and must be of type: 'string'.");
-    }
+    throwTypeErrorIfParameterMissing("connectionString", connectionString);
+    connectionString = String(connectionString);
```

Comment:
We can skip the parameter missing check here.
If none was provided, connectionString will be `undefined` and `parseConnectionString` will give appropriate errors